src/views/datatable110.blade.php

```diff
@@ -4,12 +4,12 @@
         oTable = jQuery('#{{ $id }}').DataTable({
             "processing": true,
             "serverSide": true,
-            "ajax": "/",
+            "ajax": "{{ isset($options['route']) ? $options['route'] : '/' }}",
             "columns": [
                 @foreach($columns as $name => $label)
-                { 'data': '{{ $label }}' },
+                { 'data': '{{ $name }}' },
```

Comment:
When its going through the data it uses this to define what the index's of the data is, so when you use label it looks for lets say `#` for id instead of `id`. Changing it fixes it so that label is only used in the header and the data date properly maps to the right columns.